pkg/kubectl/kubectl.go

```diff
@@ -30,6 +30,11 @@ import (
 
 const (
 	kubectlAnnotationPrefix = "kubectl.kubernetes.io/"
+	HiddenPodsWarning       = `
+WARN: By default only active pods are displayed, to display terminated
```

Comment:
This message needs to be more generic, and doesn't need the warning prefix.  It really belongs directly inside of the get comment.

```
This command will hide by default some resources that are being terminated, like pods. You can see the full results for any resource by passing the `--show-all` flag.
```

Should be a separate paragraph and in get.go in the help text.